hwcomposer/qeglfsscreen.cpp

```diff
@@ -57,7 +73,21 @@ QEglFSScreen::QEglFSScreen(HwComposerContext *hwc, EGLDisplay dpy)
 
 QEglFSScreen::~QEglFSScreen()
 {
+#ifdef WITH_SENSORS
+    if(m_orientationSensor) {
```

Comment:
A small complaint about formatting, all other files have format "if (something)" so space after if and no spaces around the content inside (). Same switches. A lot of formatting mismatch in the new changes.